hs_tools_resource/admin.py

```diff
@@ -4,5 +4,13 @@
 from mezzanine.pages.admin import PageAdmin
 
 from .models import ToolResource
+from copy import deepcopy
 
-admin.site.register(ToolResource, PageAdmin)
+author_extra_fieldsets = ((None, {"fields": ("approved",)}),)
+
+class ToolAdmin(PageAdmin):
```

Comment:
In the current state of the mezzanine admin panel, accessing resources is a pain. It would be better if an admin user can set the approved flag of any web app resources from its landing page.